docs/general/endpoints.rst

```diff
@@ -156,42 +151,29 @@ models.
 
 By this restriction, additional _features_ are calculated and returned:
 
-- ``error-transformed``:
-    - Error per feature/tag calculated from the transformed values
-- ``error-untransformed``:
-    - Error per feature/tag calculated from the ``inverse-transformed-model-output`` against ``original-input``
-- ``inverse-transformed-model-output``:
-    - The ``model-output`` which is ran backward through pipeline, which "untransforms" it to the same level
-      as the ``original-input``; this assumes that all steps in the pipeline support ``.inverse_transform``
-- ``total-transformed-error``:
-    - This is the vector norm of the error between a sample output and the sample input, using the transformed values
-- ``total-untransformed-error``:
-    - This is the vector norm of the error between a sample output and the sample input, using the inverse-transformed values
+- ``tag-anomaly``:
+    - Anomaly per feature/tag calculated from the expected tag input (y) and the model's output for those tags (yhat)
+- ``total-anomaly``:
+    - This is the total anomaly for the given point as calculated by the model.
 
 Sample response:
 
 .. code-block:: python
 
     {'data': [{'end': [None],
-               'error-transformed': [2.746687859183499,
```

Comment:
But the diffbased anomaly detector can calculate errors both the scaled and unscaled right (by using the inverse tranformed), should we do that?

Reply:
Diffbased anomaly detector doesn't have access to the pipeline, but I guess you're talking about calculating the error _without_ the `scaler` operating on the output/y as well?

Reply:
if we _do_ that, do we also update grafana? Because if we need to do that, then other anomaly models might not have such a concept, no?

Reply:
We can, but don't have to, update grafana. Whatever is easiest at this point. But the data will be there, so its easy to add if someone wants to show.

Reply:
But yeah, all I expect the diffbased detector to do is to report the errors scaled and unscaled. Although I dont know if it is actually interesting, but I seem to remember that someone wanted that earlier. But maybe what they mostly wanted was the model output in unscaled fashion? And that we get now.

Reply:
Created this issue for it; should we see use in the future: https://github.com/equinor/gordo-components/issues/392